apps/src/templates/sectionsRefresh/SingleSectionSetUp.jsx

```diff
@@ -33,6 +33,7 @@ export default function SingleSectionSetUp({
         values={section.grades || []}
         setValues={g => updateSection('grades', g)}
       />
+      <hr />
```

Comment:
was this a merge issue, or did you put it back intentionally? i remember taking this out when we decided to not do multiple sections at once, so i just wanted to check

Reply:
Ohh... I did NOT put it back intentionally... so I am guessing it is a merge issue...

Reply:
When working on this, I had a couple of issues with merging staging with my branch, but honestly, this one doesn't look familiar to me - I don't remember resolving it by hand, which makes me a bit nervous.  I might need to back up and check to see if anything else got overwritten...

Reply:
I ended up removing these by hand.